pay-api/src/pay_api/services/internal_pay_service.py

```diff
@@ -109,6 +109,11 @@ def process_cfs_refund(self, invoice: InvoiceModel):
         if routing_slip := RoutingSlipModel.find_by_number(routing_slip_number):
             routing_slip.remaining_amount += decimal.Decimal(invoice.total)
             routing_slip.flush()
+            invoice.invoice_status_code = InvoiceStatus.REFUND_REQUESTED.value
+        else:
```

Comment:
If it's not new FAS routing slip, we don't support refund. So it should be raising error, correct ?

Reply:
Good catch.Done